### DISCUSSION TITLE
<discussion> Unable to run tutorial successfully in wsl2

### DISCUSSION THREAD

**Discussion by hml1996-fight** [tensorchord/envd#1832] [Q&A]
### build.envd:
`
def build():
    base(os="ubuntu20.04", language="python3")
`
system: ### 
wsl2 - ubuntu20.04
### info:
![image](https://github.com/tensorchord/envd/assets/72264324/bf49c6f8-164b-4360-90d6-b7db9540d24b)
![image](https://github.com/tensorchord/envd/assets/72264324/8417c790-28ef-4746-994b-7ad93ae37e76)

how to fix it?

**Comment by gaocegege** [comment]
Could you please show your `docker ps -a`?

**Reply by hml1996-fight** [reply]
![image](https://github.com/tensorchord/envd/assets/72264324/c4bf1fb2-945a-4eb3-b188-6e536c306069)

**Reply by gaocegege** [reply]
It shows that your buildkitd container is created 1 minute ago. Could you please give a new run of `envd up`?

**Reply by hml1996-fight** [reply]
I just run `envd up` 1 minute ago in wsl2.
And i rerun `envd up` right now the info is:
![image](https://github.com/tensorchord/envd/assets/72264324/a4fe7d1e-f38e-4af4-af08-a1ad0a2118da)
and the build.envd is:
![image](https://github.com/tensorchord/envd/assets/72264324/3dd05884-eb85-4f3e-b143-42aeb3e25a38)

**Reply by gaocegege** [reply]
Seems like a network issue, I will give it a try.

**Reply by gaocegege** [reply]
I cannot reproduce it. But I guess it may be related to your network config. Are you using http proxy in your docker engine?

**Reply by hml1996-fight** [reply]
Maybe it's a WSL2 network error. I will check it when i free.
